Energy Saver/Services/ChartService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using ChartJSCore.Helpers;
+using ChartJSCore.Models;
+using Energy_Saver.Model;
+
+namespace Energy_Saver.Services
+{
+    public class ChartService : IChartService
+    {
+        public Chart CreateChart()
+        {
+            Chart chart = new Chart();
+            List<List<Taxes>> tableData = Serialization.ReadFromFile();
+            chart.Type = Enums.ChartType.Line;
+
+            Data data = new Data();
+            data.Labels = new List<string>() { "Gas", "Electricity", "Water", "Heating" };
+
+            LineDataset dataset = new LineDataset()
+            {
+                Label = "Amount payed all time",
```

Comment:
Should be "paid" instead of "payed". Could replace sentence with "Total paid amount"

Reply:
Fixed the typo